src/modules/commons/types.ts

```diff
@@ -0,0 +1,31 @@
+// Common types
+
+const get = {
+	start: (modelName: string) => `${modelName.toUpperCase()}_GET_MODEL`,
```

Comment:
I prefer name.GET_MODEL. Make it lowercase so we can spot it easily

 `${modelName}.GET_MODEL`